packages/excalidraw/element/binding.ts

```diff
@@ -625,8 +625,7 @@ export const updateBoundElements = (
         if (
           bindableElement &&
           isBindableElement(bindableElement) &&
-          (bindingProp === "startBinding" || bindingProp === "endBinding") &&
-          changedElement.id === element[bindingProp]?.elementId
+          (bindingProp === "startBinding" || bindingProp === "endBinding")
```

Comment:
This change will run updateBoundPoint(...) for all scene elements, which is a large amount of no-op for every movement of even a single arrow connected shape. Instead you should set up the condition that either the changed bindable element is true or the connected other end element.